core.mk

```diff
@@ -24,7 +24,7 @@ $(foreach p, $(CACHE_SUBMODULES), $(eval $p_DIR ?=$(CACHE_SUBMODULES_DIR)/$p))
 
 
 #RULES
-corename:
+$(CORE_NAME)_corename:
```

Comment:
why not 

$(CORE_NAME)_printname: 

for the target

Reply:
I just solved the warning. The target was called corename and I added $(CORE_NAME)_ as prefix. But yes, it can be changed to $(CORE_NAME)_printname.

Reply:
so change it and I will merge

Reply:
Done!